packages/commonwealth/client/scripts/views/components/react_quill_editor/react_quill_editor.tsx

```diff
@@ -247,6 +255,47 @@ const ReactQuillEditor = ({
 
   const showTooltip = isDisabled && isHovering;
 
+  const handleAddLink = () => {
+    if (linkText === '' || linkUrl === '') {
+      return;
+    }
+    const editor = editorRef?.current?.getEditor();
+    if (!editor) {
+      return;
+    }
+    // Retrieve the stored selection from childInputRef
+    const selection = childInputRef.current;
+    if (!selection) {
+      return;
+    }
+    // Format the link to ensure it has 'https://'
+    let newLink = linkUrl;
+    if (!linkUrl.startsWith('https://')) {
+      if (linkUrl.startsWith('http://')) {
+        newLink = `https://${linkUrl.substring('http://'.length)}`;
+      } else {
+        newLink = `https://${linkUrl}`;
+      }
+    }
+    // Insert the markdown-formatted link
+    const linkMarkdown = `[${linkText}](${newLink})`;
+
+    // Replace the selected text with the markdown link
+    editor.deleteText(selection.index, selection.length);
+    editor.insertText(selection.index, linkMarkdown);
+    // Set the cursor right after the inserted text
+    editor.setSelection(selection.index + linkMarkdown.length, 0);
```

Comment:
Please delete all of these comments in handleAddLink. We usually only leave comments for functions or code that isn't quickly apparent as to what they do, or if it seems unintuitive but is necessary